controllers/types.go

```diff
@@ -15,3 +26,118 @@ type PorterClient interface {
 type ClientConn interface {
 	Close() error
 }
+
+var GrpcDeployment = &appsv1.Deployment{
+	ObjectMeta: metav1.ObjectMeta{
+		Name:      PorterGRPCName,
+		Namespace: PorterNamespace,
+		Labels: map[string]string{
+			"app": "porter-grpc-service",
+		},
+	},
+	Spec: appsv1.DeploymentSpec{
+		Replicas: ptr.To(int32(1)),
+		Selector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"app": "porter-grpc-service",
+			},
+		},
+		Template: corev1.PodTemplateSpec{
+			ObjectMeta: metav1.ObjectMeta{
+				Labels: map[string]string{
+					"app": "porter-grpc-service",
+				},
+			},
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:  "porter-grpc-service",
+						Image: "ghcr.io/getporter/server:v1.1.0",
```

Comment:
Can me make the image version a const? I think it's fine to have the server version compiled into the operator controller to ensure compatibility but it should be easy to upgrade when we want to release a new version

Reply:
I can update this.